modules/apigee_edge_apidocs/tests/src/Functional/ApiDocsAdminTest.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+/**
+ * Copyright 2019 Google Inc.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * version 2 as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+ * MA 02110-1301, USA.
+ */
+
+namespace Drupal\Tests\apigee_edge_apidocs\Functional;
+
+use Drupal\file\Entity\File;
+use Drupal\Tests\BrowserTestBase;
+use Drupal\Tests\TestFileCreationTrait;
+
+/**
+ * Simple test to ensure that main page loads with module enabled.
+ *
+ * @group apigee_edge
+ * @group apigee_edge_apidocs
+ */
+class ApiDocsAdminTest extends BrowserTestBase {
+
+  use TestFileCreationTrait;
+
+  /**
+   * Modules to enable.
+   *
+   * @var array
+   */
+  public static $modules = ['apigee_edge_apidocs', 'block', 'field_ui'];
+
+  /**
+   * A user with permission to administer site configuration.
+   *
+   * @var \Drupal\user\UserInterface
+   */
+  protected $adminUser;
+
+  /**
+   * Set up menus and tasks in their regions.
+   *
+   * Since menus and tasks are now blocks, we're required to explicitly set them
+   * to regions.
+   */
+  protected function setupMenus() {
+    $this->drupalPlaceBlock('local_actions_block', ['region' => 'content']);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+    // Add the system menu blocks to appropriate regions.
+    $this->setupMenus();
+
+    $this->adminUser = $this->drupalCreateUser([
+      'add apidoc entities',
+      'delete apidoc entities',
+      'edit apidoc entities',
+      'view published apidoc entities',
+      'view unpublished apidoc entities',
+      'administer apidoc entities',
+      'administer apidoc display',
+      'administer apidoc fields',
+      'administer apidoc form display',
+      'access administration pages',
+      'access content',
```

Comment:
Why this permission is needed?

Reply:
Someone added the comment.

>       // Access content is needed to access the referenced files.

Without `access content` you get the error: "You do not have access to reference..."